nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/bin/nifi.sh

```diff
@@ -118,6 +118,12 @@ locateJava() {
             fi
         fi
     fi
+    [ "x${TOOLS_JAR}" =  "x" ] && [ -n "${JAVA_HOME}" ] && TOOLS_JAR=$(find "${JAVA_HOME}" -name "tools.jar")
```

Comment:
This was okay on OS X where my Java home was absolute, but not okay on Linux where this was a symlink.  Quick win here would be to use the -H of find